src/components/Header.js

```diff
@@ -1,13 +1,27 @@
 import React, { Component } from 'react';
 
 class Header extends Component {
+	constructor(props) {
+		super(props);
+		
+		this.keyPress = this.keyPress.bind(this);
+	}
+	
+	keyPress(e) {
```

Comment:
If you wanted to you could also check to make sure value isn't an empty string.  I also used `trim` to strip whitespace from left and right.